playwright/e2e/request-baseline.spec.ts

```
@@ -0,0 +1,189 @@
+  { name: "IAM (User Access)", url: "/iam/user-access/overview" },
+  { name: "IAM (My Access)", url: "/iam/my-user-access" },
+];
+
+test.describe("Akamai Request Baseline", () => {
+  // Run all pages even if one fails — we want the full baseline, not early exit.
+  test.describe.configure({ mode: "parallel" });
+
+  for (const landing of LANDING_PAGES) {
+    test(`${landing.name} — request count baseline`, async ({ page }) => {
+      const requests: CapturedRequest[] = [];
+      const origin = new URL(page.url() || "https://stage.foo.redhat.com:1337").origin;
```

ESLint

[error] Replace `"https://stage.foo.redhat.com:1337"` with `'https://stage.foo.redhat.com:1337'`

(prettier/prettier)